code/datums/components/fantasy/suffixes.dm

```diff
@@ -276,3 +276,17 @@
 /datum/fantasy_affix/speed/remove(datum/component/fantasy/comp)
 	var/obj/item/master = comp.parent
 	master.slowdown = initial(master.slowdown)
+
+/datum/fantasy_affix/doot
+	name = "of dooting"
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD
+	weight = 1
+
+/datum/fantasy_affix/doot/apply(datum/component/fantasy/comp, newName)
+	. = ..()
+	comp.parent.AddElement(/datum/element/spooky, too_spooky = comp.quality > 17, stam_damage_mult = comp.quality * 0.15)
+	return "[newName] of [pick("dooting", "spooks", "rattling", "the bones")]"
+
+/datum/fantasy_affix/doot/remove(datum/component/fantasy/comp)
+	comp.parent.RemoveElement(/datum/element/spooky, too_spooky = comp.quality > 17, stam_damage_mult = comp.quality * 0.15)
```

Comment:
nothing changes comp quality yeah? cause that'd be bad here

Reply:
I was checking this myself because I almost posted the same review, but as far as I can tell it calls `remove` before any change to `comp.quality` and then reapplies it afterwards. Worth double checking from the developer though 😛

Reply:
Sorry for the late reply but yes, if you are changing the quality correctly through component inheritance the various effects will be removed and then reapplied with the new values, so it shouldn't be a problem.